src/components/Header.vue

```diff
@@ -1,3 +1,64 @@
 <template>
-    <header>My Header</header>
-</template>
\ No newline at end of file
+    <header>
+        <div class="header-content">
+            <Navbar/>
```

Comment:
```suggestion
            <navbar></navbar>
```

C'est mieux dans le template d'appeler les composants comme ça c'est une convention Vue :)

Reply:
Je suis bien à jour avec main Lucas, j'ai normalement réglé tous les confilts.
Je note pour la convention Vue :-)

ça marche pour la navbar <navbar></navbar>
mais dans App.  ni <footer></footer> ni <header></header> ne fonctionnent.  tu saurais pourquoi ?

Sinon l'intégration de base est en place :-)